src/test.js

```diff
@@ -485,17 +485,7 @@ function synchronize( callback, priority ) {
 
 // Place previously failed tests on a queue priority line, respecting the order they get assigned.
 function priorityFill( callback ) {
-	var queue, prioritizedQueue;
-
-	queue = config.queue.slice( priorityFill.pos );
-	prioritizedQueue = config.queue.slice( 0, -config.queue.length + priorityFill.pos );
-
-	queue.unshift( callback );
-	queue.unshift.apply( queue, prioritizedQueue );
-
-	config.queue = queue;
-
-	priorityFill.pos += 1;
+	config.queue.splice( priorityFill.pos++, 0, callback);
```

Comment:
Do we even still need a separate function? Looks like you could just inline the whole thing now.